datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -1054,6 +1061,44 @@ private void validateSdxResizeRequest(SdxCluster sdxCluster, String accountId, S
                 });
     }
 
+    private void validateDatalakeNotInBackupOrRestore(String sdxClusterName, String actorCrn) {
+        if (sdxClusterName == null || actorCrn == null) {
+            LOGGER.error("Didn't find datalake name or actor CRN. Can not validate if the datalake is in backup/restore. Continue to next step.");
+            return;
+        }
+
+        DatalakeBackupStatusResponse backupStatus = null;
+        DatalakeRestoreStatusResponse restoreStatus = null;
+        try {
+            backupStatus = datalakeDrClient.getBackupStatus(sdxClusterName, actorCrn);
+            restoreStatus = datalakeDrClient.getRestoreStatus(sdxClusterName, actorCrn);
+        } catch (RuntimeException exception) {
+            Pattern pattern = Pattern.compile("Status information for backup|restore operation on datalake.*not found");
+            Matcher matcher = pattern.matcher(exception.getMessage());
+            if (matcher.find()) {
+                LOGGER.info("No backup or no restore ever happened on {}. Good to continue.", sdxClusterName);
+            } else {
+                throw exception;
```

Comment:
Add some logging here as this is a weird scenario.

Reply:
@sxxgrc I added some comments in there but not sure if that's what you want. Please let me know if it looks good. Thank you!!